src/reformatters/noaa/gfs/forecast/template_config.py

```diff
@@ -0,0 +1,635 @@
+            calendar="proleptic_gregorian",
+            units="seconds since 1970-01-01 00:00:00",
+            chunks=(INIT_TIME_COORDINATE_CHUNK_SIZE, len(_dim_coords["lead_time"])),
+            shards=(INIT_TIME_COORDINATE_CHUNK_SIZE, len(_dim_coords["lead_time"])),
+        ),
+        attrs=CoordinateAttrs(
+            units="seconds since 1970-01-01 00:00:00",
+            statistics_approximate=StatisticsApproximate(
+                min=INIT_TIME_START.isoformat(), max="Present + 16 days"
+            ),
+        ),
+    ),
+    Coordinate(
+        name="ingested_forecast_length",
+        encoding=Encoding(
+            dtype="int64",
+            fill_value=-1,
+            compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
+            units="seconds",
+            chunks=(INIT_TIME_COORDINATE_CHUNK_SIZE, len(_dim_coords["lead_time"])),
+            shards=(INIT_TIME_COORDINATE_CHUNK_SIZE, len(_dim_coords["lead_time"])),
+        ),
+        attrs=CoordinateAttrs(
+            units="seconds",
+            statistics_approximate=StatisticsApproximate(
+                min=str(_dim_coords["lead_time"].min()),
+                max=str(_dim_coords["lead_time"].max()),
+            ),
+        ),
+    ),
+    Coordinate(
+        name="expected_forecast_length",
+        encoding=Encoding(
+            dtype="int64",
+            fill_value=-1,
+            compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
+            units="seconds",
+            chunks=INIT_TIME_COORDINATE_CHUNK_SIZE,
+            shards=INIT_TIME_COORDINATE_CHUNK_SIZE,
+        ),
+        attrs=CoordinateAttrs(
+            units="seconds",
+            statistics_approximate=StatisticsApproximate(
+                min=str(_dim_coords["lead_time"].min()),
+                max=str(_dim_coords["lead_time"].max()),
+            ),
+        ),
+    ),
+    Coordinate(
+        name="spatial_ref",
+        encoding=Encoding(
+            dtype="int64",
+            fill_value=0,
+            chunks=1,  # Scalar coordinate
+            shards=1,
+        ),
+        attrs=CoordinateAttrs(
+            units="unitless",
+            statistics_approximate=StatisticsApproximate(
+                min=0,
+                max=0,
+            ),
+        ),
+    ),
+)
+
+# index_position = BAND - 1 from the grib files
+DATA_VARIABLES: Sequence[GFSDataVar] = (
+    GFSDataVar(
+        name="pressure_surface",
+        encoding=replace(ENCODING_FLOAT32_DEFAULT),
+        attrs=DataVarAttrs(
+            short_name="sp",
+            long_name="Surface pressure",
+            units="Pa",
+            step_type="instant",
+            standard_name="surface_air_pressure",
+        ),
+        internal_attrs=GFSInternalAttrs(
+            grib_element="PRES",
+            grib_description='0[-] SFC="Ground or water surface"',
+            grib_index_level="surface",
+            index_position=560,
+            keep_mantissa_bits=8,
```

Comment:
```suggestion
            keep_mantissa_bits=10,
```
I made this too low in GEFS, going to increase it there too. the values are in a relatively small range of all very large values so we need more bits (because float numbers become more sparse/quantized the farther from  zero).